CMakeLists.txt

```diff
@@ -78,7 +79,6 @@ else()
   add_cxx_compiler_flag(-pedantic-errors)
   add_cxx_compiler_flag(-Wshorten-64-to-32)
   add_cxx_compiler_flag(-Wfloat-equal)
-  add_cxx_compiler_flag(-Wzero-as-null-pointer-constant)
```

Comment:
maybe comment it out with this as documentation? unless you think there's never a time we'll be able to put it back.